### PR TITLE
[WFLY-17947]: todo-backend Readme OpenShift instructions results in a…

### PR DESCRIPTION
**Pull request by ehsavoie**
… non-functional QS app.

* Fixing docs and pulling back composite helm charts.

Jira: https://issues.redhat.com/browse/WFLY-17947

### CODE REVIEW

File: todo-backend/todo-backend-chart/Chart.lock [hunk 1]
```diff
@@ -0,0 +1,9 @@
+dependencies:
```
Comment (zhantemirov) [inline todo-backend/todo-backend-chart/Chart.lock:1]: Does this file need to be in commit?

**Comment by zhantemirov** [comment]
Do .tgz files need to be in this PR?

**Comment by zhantemirov** [comment]
The Readme should be reworked, I've prepared the changes that can be cherry-picked https://github.com/zhantemirov/quickstart/commits/readme-fixes

File: todo-backend/charts/helm.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ build:
   contextDir: todo-backend
   env:
     - name: POSTGRESQL_DRIVER_VERSION
-      value: '42.5.1'
```
[the remark this answers is not in the document]
Comment (ehsavoie) [reply todo-backend/charts/helm.yaml:9]: I don't know

**Comment by fjuma** [comment]
@ehsavoie Since WFLY-17947 also has the 28.0.1.Final fix version, we'll also need a PR for this against the 28.x branch.

**Comment by ehsavoie** [comment]
@fjuma this should be ok now
@zhantemirov could you please ack the latest push ?

File: jaxrs-client/pom.xml [hunk 3]
```diff
@@ -287,6 +287,10 @@
                             <layers>
                                 <layer>cloud-server</layer>
                             </layers>
+                            <!-- Workaround to avoid opentelemetry issue that breaks the Jaxrs applications: removed once WFLY-17955 is integrated -->
```
Comment (emmartins) [inline jaxrs-client/pom.xml:290]: I see that WFLY-17955 is resolved and fix will be on upcoming releases, do we still need these workarounds?

Comment (ehsavoie) [reply jaxrs-client/pom.xml:290]: I depends on if we target only 28.0.1 or all 28 releases

Comment (emmartins) [reply jaxrs-client/pom.xml:290]: I mean QS will provisioning either 29.0.0 Alpha or 28.0.1 Final, depending on branch, and both seem to have the fix.

**Comment by fjuma** [comment]
> @fjuma this should be ok now 

Thanks @ehsavoie! When you get a chance, please also create a PR against the 28.x branch and let @emmartins know.

**Review by zhantemirov — APPROVED** [review]

**Comment by emmartins** [comment]
I merged this after @zhantemirov approval, I will do any changes (like removing the workarounds once wildfly is released and upgraded on the QS), thanks all

**Pull request merged**